handling/aconnect.go

```diff
@@ -25,6 +25,11 @@ type AtlassianConnect struct {
 	Name           string                 `json:"name,omitempty"`
 	Scopes         []string               `json:"scopes,omitempty"`
 	Vendor         Vendor                 `json:"vendor,omitempty"`
+	APIMigrations  APIMigration           `json:"apiMigrations,omitempty"`
+}
+
+type APIMigration struct {
+	SignedInstall bool `json:"signed-install"`
```

Comment:
any reason for camel case outside and dashes inside?

Reply:
you will have to ask atlassian :p it's their schema

Reply:
"deus vult".